src/components/LoginForm/LoginForm.jsx

```diff
@@ -37,11 +40,20 @@ const LoginForm  = () =>{
                         token: response.token,
                     });
                     navigate("/events");
```

Comment:
Might be a good idea to navigate to /profile on login, now that we have a profile page